maven/src/org/netbeans/modules/maven/api/Constants.java

```diff
@@ -150,6 +150,15 @@
      * @since 2.113
      */
     public static final String ACTION_PROPERTY_JPDAATTACH_TRANSPORT = "jpda.attach.transport";
+ 
+    /**
+     * Name of a property used for checking the maven output for the approriate time to attach to
+     * {@link #ACTION_PROPERTY_JPDAATTACH_ADDRESS} address.
+     * This property is meaningful only when {@link #ACTION_PROPERTY_JPDAATTACH_ADDRESS}
+     * is defined.
+     * @since 2.114
```

Comment:
I don't see associated change to `manifest.mf`, is it there? The correct approach is to increase the version in the manifest and then let `@since` refer to it.